From ec9c065d405a129d3d33abd3d10d8a7ff1a49420 Mon Sep 17 00:00:00 2001
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Wed, 18 Jan 2017 10:48:12 +0000
Subject: [PATCH] Disable use of namespaces by default

When namespaces are enabled there is currently breakage when
using disk hotplug and when using AppArmor

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
---
 src/qemu/qemu_conf.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/qemu/qemu_conf.c b/src/qemu/qemu_conf.c
index 6613d59..aa05b46 100644
--- a/src/qemu/qemu_conf.c
+++ b/src/qemu/qemu_conf.c
@@ -317,13 +317,6 @@ virQEMUDriverConfigPtr virQEMUDriverConfigNew(bool privileged)
     if (!(cfg->namespaces = virBitmapNew(QEMU_DOMAIN_NS_LAST)))
         goto error;
 
-#if defined(__linux__)
-    if (privileged &&
-        virProcessNamespaceAvailable(VIR_PROCESS_NAMESPACE_MNT) == 0 &&
-        virBitmapSetBit(cfg->namespaces, QEMU_DOMAIN_NS_MOUNT) < 0)
-        goto error;
-#endif /* defined(__linux__) */
-
 #ifdef DEFAULT_LOADER_NVRAM
     if (virFirmwareParseList(DEFAULT_LOADER_NVRAM,
                              &cfg->firmwares,
-- 
2.10.2

